src/adaptors/sense/index.js

```diff
@@ -0,0 +1,289 @@
+    .div(timeSinceInit / ONE_DAY_SECONDS)
+    .times(ONE_YEAR_DAYS);
+  // Space fees are taken as a percent of yield, so use the current PT implied rate as the base yield from which to take.
+  // Then, determine the percent of the current pool our estimated yearly volume is (even if the pool won't be around for a year,
+  // since we're calculating apy) and use that to weight the avg additional underlying we collect per underlying traded.
+  const estYieldFromFees = new BigNumber(pool.impliedRate)
+    .div(100)
+    .times(0.05)
+    .times(estVolumePerYear.div(total));
+
+  const targetTvl = new BigNumber(pool.targetBalance).times(
+    targetPrices[pool.series.target]
+  );
+  const ptTvl = BigNumber(pool.pTBalance)
+    .times(targetPerPT)
+    .times(targetPrices[pool.series.target]);
+
+  return [
+    // Take the apy as the weighted sum of the Target and PT APYs, plus the estimated yield from fees.
+    weightedPTApy.plus(weightedTargetApy).plus(estYieldFromFees).toString(),
+    targetTvl.plus(ptTvl).toString(),
+  ];
+};
+
+const main = async () => {
+  let { pools } = await request(graphEndpoint, query);
+
+  const yields = (await superagent.get('https://yields.llama.fi/pools')).body
+    .data;
+
+  pools = pools.map((pool) => {
+    const y = yields
+      .filter((y) => y.symbol === targetToTicker(pool.series.targetName))
+      .reduce(
+        (acc, cur) =>
+          acc === null ? cur : cur.tvlUsed > acc.tvlUsed ? cur : acc,
+        null
+      );
+
+    if (y == null) return pool;
+
+    return { targetAPYBase: y.apyBase, targetAPY: y.apy, ...pool };
+  });
+
+  const targetTokens = [...new Set(pools.map((pool) => pool.series.target))];
+  const targetPrices = await getPrices(targetTokens);
+
+  const ptAdapterMap = {};
+  for (const { series } of pools) {
+    const { pt, adapter } = series;
+    ptAdapterMap[pt] = adapter;
+  }
+
+  const adapterDecimalsMap = {};
+  for (const { series } of pools) {
+    const { decimals, adapter } = series;
+    adapterDecimalsMap[adapter] = decimals; // Decimals are constant for all Series of a particular adapter
+  }
+
+  const ptTotalSupplys = new Map(
+    await Promise.all(
+      Object.entries(ptAdapterMap).map(async ([ptToken, adapterAddress]) => [
+        ptToken,
+        new BigNumber(
+          (
+            await sdk.api.erc20.totalSupply({
+              target: ptToken,
+              chain: 'ethereum',
+            })
+          ).output
+        )
+          .div(10 ** parseInt(adapterDecimalsMap[adapterAddress]))
+          .toString(),
+      ])
+    )
+  );
+
+  // SENSE PTS
+  const pts = pools.map((pool) => ({
+    pool: pool.series.pt,
+    chain: 'Ethereum',
+    project: 'sense',
+    symbol: `PT-${underlyingToTicker(pool.series.underlyingName)} ${
+      pool.series.targetName
+    }-${pool.series.adapterMeta.number} (Maturing ${toISODate(
+      pool.series.maturity
+    )})`,
+    tvlUsed: new BigNumber(ptTotalSupplys.get(pool.series.pt))
```

Comment:
```suggestion
    tvlUsd: new BigNumber(ptTotalSupplys.get(pool.series.pt))
```

Reply:
oop, I corrected this then flipped it back. Will fix

Reply:
fixed